bundle-workflow/README.md

```diff
@@ -120,6 +120,15 @@ The following options are available.
 
 This step runs integration tests invoking `integtest.sh` in each component from bundle manifest.
 
+To run integration tests locally, use below command. It pulls down the built bundle and its manifest file from S3,
+reads all components of the bundle and runs integration tests against each component.
+ 
+```
+cd opensearch-build/bundle-workflow
+pipenv install
```

Comment:
It's worth making this into test.sh earlier than later even if it's a little hacky, because this is effectively the calling interface.

It should be `test.sh --suite ... --S3..` and would make it callable from everywhere and no need to remember to use pipenv.

Reply:
Same comment as dB. The runner should install all the python dependencies. I did see problems when the Jenkins job was being setup.

Reply:
Agree. It will be taken as a separate PR after this.

Reply:
Please consider doing it as part of this PR? It's a tiny amount of work. This is just not reliable.

Reply:
I have these basic changes in my PR. Feel free to use them or I could push it as part of my PR.
https://github.com/opensearch-project/opensearch-build/pull/423/files#diff-df03dd40a41319c80f83f3d4c9cde02ba41aed2d9d102e04546e9b8c1c5dc5ae

Reply:
We have to revisit this part when make the input arguments uniform across all test jobs. Currently, BwC accepts different input parameters and so does the perf test. The plan was to do this unification and shell-ify as part of that single task, as it is all related. https://github.com/opensearch-project/opensearch-build/pull/456#discussion_r707860346
This doesn't seem to be a blocker for the purpose of this PR. A separate issue to track this #472.

Reply:
I disagree with punting shellifying to the future. It's a trivial change that ensures that runs are predictably reliable. No need to acquire more debt.

Reply:
Added this